### DISCUSSION TITLE
Is it possible to build chat app using deta micros?

### DISCUSSION THREAD

**Discussion by adipdwi** [deta/cloud-docs#751] [Help] [answered]
Is it possible to build chat app using deta micros? I don't have any idea if it possible or not

**Comment by tongsangcong** [comment, marked as answer]
The chat app is usually built using websocket to give a "real time" interaction and deta micros is not supporting websocket. you could refer the documentation here https://docs.deta.sh/docs/micros/faqs_micros#do-micros-support-websockets
